test/e2e/e2e_ginkgo/subscription_e2eg_test.go

```diff
@@ -0,0 +1,16 @@
+package e2e_ginkgo
+
+import (
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+)
+
+var (
+	strategy = v1alpha1.StrategyDetailsDeployment{
+		DeploymentSpecs: []v1alpha1.StrategyDeploymentSpec{
+			{
+				Name: genName("dep-"),
+				Spec: newNginxDeployment(genName("nginx-")),
+			},
+		},
+	}
+)
```

Comment:
nit: do have extra new line at the end of the file.

Reply:
done